extensions/ql-vscode/src/extension.ts

```diff
@@ -462,7 +462,7 @@ export async function activate(
     ) {
       registerErrorStubs([checkForUpdatesCommand], (command) => async () => {
         const installActionName = "Install CodeQL CLI";
-        const chosenAction = await void showAndLogErrorMessage(
+        const chosenAction = await showAndLogErrorMessage(
```

Comment:
Interesting. I guess it was never possible to install the cli from this code path. I never tried this out, though so I don't really know.

Thanks for catching this.